backend/project404_t8/API/viewsets_sql_query.sql

```diff
@@ -0,0 +1,45 @@
+-- 1: selected>Just Me
```

Comment:
file explaining everything being done with the SQL query. this is so that we all understand it and make changes to other parts copied and pasted from here